sunpy/instr/goes.py

```diff
@@ -69,8 +69,11 @@
 from sunpy.util.net import check_download_file
 from sunpy import sun
 
+GOES_CONVERSION_DICT = {-4: 'X', -5: 'M', -6: 'C', -7: 'B', -8: 'A'}
```

Comment:
I think, it will make it easier to debug code if an error shows that dictionary...